src/S3Policy.js

```diff
@@ -35,7 +35,9 @@ export class S3Policy {
 }
 
 const getDate = () => {
-  let date = new Date();
+  let date = new Date(
```

Comment:
Yea I did, I was mistaken on the requirements for dates when generating a signature. The only thing that actually had to match current time is the expiration date.